docs/guide/rpc-api.md

```diff
@@ -178,8 +177,6 @@ function requestPermissions() {
 }
 ```
 
-## Other RPC Methods
-
```

Comment:
I believe that the unrestricted methods section should start here.

Reply:
See the note in the PR description:

>The decryption methods were pulled into the "restricted methods" section because they are restricted methods.

Reply:
Heretofore I've defined "restricted methods" as a method that has a permission, not a method that has a confirmation. The decryption methods are in our unrestricted methods constants that are passed to the permission controller.

Reply:
Ref: https://github.com/MetaMask/metamask-extension/blob/develop/app/scripts/controllers/permissions/specifications.js/#L205

Reply:
Or you mean to say that it's de facto restricted because it relies on `eth_accounts`?

Reply:
Yes exactly. It's restricted by the eth accounts permission. It returns an error without it.

Reply:
We discussed this on Slack and decided to leave them as "Unrestricted" for now, but to ensure that the definition of "Unrestricted" didn't imply that no permission was needed. Just that there was no permission with a name matching the RPC method name. Also we should add a warning explaining that the permission is needed, but that can be in a separate PR. I will update this PR.

Reply:
Addressed in https://github.com/MetaMask/metamask-docs/pull/348/commits/c42ac389bef377113968295b5537f6118df0300a, and the PR description has been updated.